templates/html-header.twig

```diff
@@ -37,6 +37,12 @@
 	{% if css_vars %}
 		{% include 'css-variables.twig' %}
 	{% endif %}
+	<script>
```

Comment:
This is just a quick fix for now to ensure the theme's properties are set early enough. It does the same thing as https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/d90243cf861c5e5909db59c546516bc6f58315cb/assets/src/theme/initializeThemeEditor.js#L37-L40 but that runs too late so it causes a flicker effect.